docs/sources/installation/docker.md

```diff
@@ -85,6 +85,8 @@ docker run \
   grafana/grafana
 ```
 
+> If you need to specify the version of a plugin, you can add it to the `GF_INSTALL_PLUGINS` environment variable. Otherwise, the latest will be assumed. For example:`-e "GF_INSTALL_PLUGINS=grafana-clock-panel 1.0.1,grafana-simple-json-datasource 1.3.5"`
+
```

Comment:
I think that it would be easier to read if you added a space after `For example:`